src/components/filters/DescFilter.tsx

```diff
@@ -0,0 +1,109 @@
+import { filterProps } from '@/pages/StatsDashboard'
+import BaseFilter from './BaseFilter'
+import { LucideCalendar, LucideChevronDown, LucideChevronRight, LucideTrash2 } from 'lucide-react'
+import { useEffect, useState } from 'react'
+import {
+  useFloating,
+  offset,
+  flip,
+  shift,
+  autoUpdate,
+  useClick,
+  useDismiss,
+  useRole,
+  useInteractions,
+  FloatingFocusManager,
+} from '@floating-ui/react'
+
+enum DescFilterOption {
+  CONTAINS = 'contiene',
+}
+const possibleDescFilterOptions: DescFilterOption[] = [
+  DescFilterOption.CONTAINS,
+]
+
+const FilterDesc = ({ id, dispatch }: filterProps) => {
+  const [search, setSearch] = useState('')
+  const [selectedDescFilter, setSelectedDescFilter] = useState(DescFilterOption.CONTAINS)
+  const [isDescFilterSelectOpen, setIsDescFilterSelectOpen] = useState(false)
+
+  useEffect(() => {
+    switch (selectedDescFilter) {
+      case DescFilterOption.CONTAINS:
+        dispatch({ type: 'UPDATE_FILTER', payload: { id: id, operation: (incident) => incident.description.includes(search) } })
```

Comment:
I wonder if you could make this case-insensitive? It doesn't seem likely that users will want to filter only by capitalization.